docs/usage/self-hosted-experimental.md

```diff
@@ -100,6 +100,14 @@ Allowed values:
 
 Default value: `asc`.
 
+## `RENOVATE_X_REBASE_PAGINATION_LINKS`
+
+<!-- prettier-ignore -->
+!!! note
+    For the GitHub Enterprise Server platform only.
```

Comment:
After you've applied my suggestion about the `endpoint`, please move the admonition block to the bottom of the section.